### ISSUE TITLE
Complete the WebAIM Document Accessibility Course (Faith)

### ISSUE THREAD

**Issue by cphillipsusu** [USU-NCADEMI/site-support#7]
You should have received an email from [notifications@instructure.com](mailto:notifications@instructure.com) inviting you to participate in a document accessibility course. While the course it is Canvas you will need to create a new account separate from your student account. The course should be mostly self-explanatory, but let me know if you have any questions. This can be taken on your own time over the next few weeks. While the focus is on documents instead of the web - you will likely be helping with accessible PDF files at some point and it provides a solid overview of accessibility that will apply to our web work. 

There is also an email from alyson.bell@usu.edu, if you need help logging into the course. 

Here is the URL to login once you. have joined the course: https://usucourses.instructure.com/courses/2802/
